pkg/agent/sysadvisor/types/types.go

```diff
@@ -140,6 +158,20 @@ type RegionEntries map[string]*RegionInfo
 // PodSet stores container names keyed by pod uid
 type PodSet map[string]sets.String
 
+// InternalCalculationResult conveys minimal information to cpu server for composing
+// calculation result
+type InternalCalculationResult struct {
```

Comment:
No in the short term, because cpu plugin only supports cpuset tweak. Even if we need to extend for more control knobs, adding more data members in InternalCalculationResult works.